LocalSettings.php

```diff
@@ -2489,12 +2489,18 @@
 			'sysadmin' => [
 				'globalgroupmembership' => true,
 				'globalgrouppermissions' => true,
+				'oathauth-api-all' => true,
+				'oathauth-disable-for-user' => true,
 			],
 			'trustandsafety' => [
 				'userrights' => true,
 				'globalgroupmembership' => true,
 				'globalgrouppermissions' => true,
 				'userrights-interwiki' => true,
+				'centralauth-lock' => true,
```

Comment:
If this is being moved for t&s then it should also be moved for sysadmin

Reply:
> If this is being moved for t&s then it should also be moved for sysadmin

Does SRE still have `centralauth-lock` globally? If _so_, then my idea against moving it for `sysadmin` is because I don't think SRE has a need for this user right. As with `userrights` and `userrights-interwiki`, they have the capability to temporarily add this user right for any emergency actions they need to take. I would recommend @Reception123 remove it from the global group

Reply:
I'm fine with that. The less rights we have by default is better.